projects/fsastorefrontlib/src/cms-components/agent/agent-search-list/agent-search-list.component.html

```diff
@@ -37,7 +37,10 @@
           </button>
         </div>
       </ng-container>
-      <div class="col-lg-6 col-12 mb-5">
+      <div
+        class="col-lg-6 col-12 mb-5 order-lg-3"
+        [class.order-2]="!searchQuery && selectedIndex == 0"
```

Comment:
Need `searchQuery` criteria, because when user choose agent from List view by clicking on the location icon, map was displayed above selected agent instead of below.

Reply:
- Couldn't we just moved the code in HTML template instead of moving it with CSS?
- `selectedIndex == 0` should have `===`

Reply:
You mean to add new template with new map? We already have 2 maps in the html, I think it would be too much. Isn't it better to use an existing map, and change position with order?

I'll change to `selectedIndex === 0`

Reply:
No, I just meant to change the order in HTML (what was left, move to right, and vice versa) instead of with CSS. Can this be done?

Reply:
Probably can, but also you'll need css classes for that case.